src/generated/resources/assets/dungeonsdelight/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+{
+  "block.dungeonsdelight.cut_stained_scrap": "切制污秽残片",
+  "block.dungeonsdelight.cut_stained_scrap_slab": "切制污秽残片台阶",
+  "block.dungeonsdelight.cut_stained_scrap_stairs": "切制污秽残片楼梯",
+  "block.dungeonsdelight.dungeon_stove": "地牢炉灶",
+  "block.dungeonsdelight.embedded_eggs": "嵌入蛋",
+  "block.dungeonsdelight.glow_berry_gelatin_block": "发光浆果凝胶块",
+  "block.dungeonsdelight.guardian_angel_block": "守护天使",
```

```suggestion
  "block.dungeonsdelight.guardian_angel_block": "守卫者填食",
```
从方块模型上可以看出原型是守卫者（Guardian），因而Guardian这部分应当保留为原版译名。Angel这部分直译为“天使”似乎不怎么像食物，可以考虑“填食”这个谐音双关，毕竟原文也是双关。

---

哦淦 忘记修改这一个双关了 感谢佬Orz